src/test/resources/features/extractMetadata/extract.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Extract Metadata
+
+  In order to extract metadata from a file
+  As a valid Converter customer
+
+  Background: Sets authentication
+    Given I set valid authentication headers
+  
+  @functional
+  Scenario: Verify that is possible to extract metadata from an file with "exportFormat" field as TXT
+    When I send a POST request to "/extractMetadata" with the following form data
+      | file         | @"template/audio/demo.mp3"        |
+      | md5          | e1b3fab24c8af81c1aa13dbbb4e44ff0 |
+      | exportFormat | t                                |
+      | fileName     | testExtractMetaData              |
+      | detail       | d                                |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "common/messageResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 200 |
+
+  @functional
+  Scenario: Verify that is possible to extract metadata from an file with "exportFormat" field as CSV
+    When I send a POST request to "/extractMetadata" with the following form data
+      | file         | @"template/audio/demo.mp3"        |
+      | md5          | e1b3fab24c8af81c1aa13dbbb4e44ff0 |
+      | exportFormat | T                                |
+      | fileName     | testExtractMetaData              |
+      | detail       | d                                |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "common/messageResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 200                                                                     |
+
+  @functional
+  Scenario: Verify that is possible to extract metadata from an file with "exportFormat" field as JSON
```

Comment:
Please consider improving this title to something like: `Verify that is possible to extract metadata from a file using JSON export format`